bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/resources/SaveManager.java

```diff
@@ -1495,27 +1496,29 @@ protected void snapTree(ElementTree tree, IProgressMonitor monitor) throws CoreE
 	}
 
 	/**
-	 * Sorts the given array of trees so that the following rules are true:
```

Comment:
This comment is outdated now, or why is this removed? It contains information we don't have anywhere else.

Reply:
1. information was wrong. As proofed with junit the first tree could have a parent.
2. the valid information extended in the param section